roles/openshift_metrics/files/templates/hawkular_cassandra_nodes_service.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: v1
```

Comment:
do we want to use a service template similar to what we use for logging with these services?